sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/RntbdTransportClientTest.java

```diff
@@ -664,7 +664,7 @@ public void verifyGoneResponseMapsToGoneException() throws Exception {
      * These are the exceptions that cannot be derived from server responses. They are mapped from Netty channel
      * failures simulated by {@link FakeChannel}.
      *
-     * @param builder   A feature validator builder to confirm that response is correctly mapped to an exception
+     * @param builder   A feature validator changeFeedProcessorBuilder to confirm that response is correctly mapped to an exception
```

Comment:
undo,

changeFeedProcessorBuilder is not relevant to rntbd.

I think you replaced build as a string with changeFeedProcessorBuilder. Please check for other instances as well. and undo

Reply:
Thank you. Corrected